tx/transaction.go

```diff
@@ -487,6 +492,43 @@ func IntrinsicGas(clauses ...*Clause) (uint64, error) {
 	return total, nil
 }
 
+// VIP-215
+func CreateFromETHTransaction(nonce uint64, chainTag byte, gasPrice *big.Int, v byte, r, s *big.Int) (*Transaction, error) {
+	if !crypto.ValidateSignatureValues(v, r, s, false) {
+		return nil, errInvalidSignature
+	}
+
+	tx:= &Transaction {
+        body: *createBody(nonce, chainTag, *gasPrice),
+    }
+
+	return tx, nil
+}
+
+// VIP-215
+func createBody(nonce uint64, chainTag byte, gasPrice big.Int) *body {
+	var gp uint64 = uint64(gasPrice.Uint64())
+    return &body {
+        Nonce: nonce,
+		ChainTag: chainTag,
+		GasPriceCoef: calcuateGasPriceCoef(gp),
+		// BlockRef     uint64
+		// Expiration   uint32
+		// Clauses      []*Clause
+		// Gas          uint64
+		// DependsOn    *thor.Bytes32 `rlp:"nil"`
+		// Reserved     reserved
+		// Signature    []byte
+		// &t.body.Reserved, // TODO?: features.contants.EthFeatures
```

Comment:
Can be now something like:
    Reserved:  reserved{Features: EthTxnFeature},

Reply:
Done.